website/docs/artist_hosts_blender.md

```diff
@@ -284,3 +284,24 @@ select **Link Animation (blend)**.
 
 Loading an animation should be done after loading the layout or rigs that have
 been animated in the animation task.
+
+## Geometry nodes
+
+Creating and publishing geometry nodes with OpenPype follows similar workflow as with
+other data types.
+
+:::note
+This family works only across blender files and cannot be compatible with any other host.
+:::
+
+### Publishing geometry nodes
+
+First of all, you must create a `Blender Nodegroup` instance from **OpenPype → Create...** which will create a collection to which you can link the objects holding you want to publish. All geometry nodes linked to an object by a `Geometry Nodes` modifier will be published.
```

Comment:
Since the target branch is related to seamless integration, I think the documentation should also explain how to use blender nodegroups with the seamless integration workflow.

Reply:
Hum, sorry for that but I've put this PR to the wrong branch...